xmtp_mls/src/utils/mod.rs

```diff
@@ -0,0 +1,5 @@
+pub mod hash;
+#[cfg(test)]
```

Comment:
I added some global test utils (mostly just moving existing ones from the store), but they are only exported in `cfg(test)`